File: C++/all_numbers_specified_digits.cpp
# include <iostream>
# include <math.h>
using namespace std;
int main () {
	int x;
	int lower;
	int upper;
	cout << "User, enter the number of digits: " << endl;
	cin >> x;
	lower = (int)(pow(10,x-1));
	upper = (int)(pow(10,x)-1);
	// cout << "The entered number is: " << x << endl;
	while (lower <= upper) {
		cout << lower << endl;
		lower++;
	}
}

File: PYTHON/all_numbers_specified_digits.py
def all_numbers_digits (dig):
	dig = int(dig)
	lower = int(pow(10,dig-1))
	upper = int(pow(10,dig)-1)
	while (lower <= upper):
		print(str(lower), end=",")
		lower = int(lower + 1)
	print("")

x = int(input("User, enter a number: "))
all_numbers_digits(x)
